### PR TITLE
RHDEVDOCS-3946 Pipelines 1.7 Tutorial + RHDEVDOCS-3977 Delete optional CRs before uninstalling Operator

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.10`, `enterprise-4.11`
- **JIRA issues**: 
  - [RHDEVDOCS-3946 Pipelines 1.7 Tutorial](https://issues.redhat.com/browse/RHDEVDOCS-3946)
  - [RHDEVDOCS-3977 Delete optional CRs before uninstalling Operator](https://issues.redhat.com/browse/RHDEVDOCS-3977)
- **Preview pages**:
  - [Creating CI/CD solutions for applications using OpenShift Pipelines](https://deploy-preview-44804--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/creating-applications-with-cicd-pipelines.html)
  - [Uninstalling OpenShift Pipelines](https://deploy-preview-44804--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/uninstalling-pipelines.html)
- **SME review**: @savitaashture @ppitonak @nikhil-thomas       
- **Peer-review**: @Srivaralakshmi

### CODE REVIEW

File: cicd/pipelines/uninstalling-pipelines.adoc [hunk 1]
```diff
@@ -6,9 +6,15 @@ include::_attributes/common-attributes.adoc[]
...
 
-Uninstalling the {pipelines-title} Operator is a two-step process:
+Cluster administrators can uninstall the {pipelines-title} Operator by performing the following steps:
 
 . Delete the Custom Resources (CRs) that were added by default when you installed the {pipelines-title} Operator.
+. Delete the CRs of the optional components, such as {tekton-chains}, that are dependent on the Operator.
++
+[CAUTION]
+====
+If you uninstall the Operator without removing the CRs of optional components, you can not remove them later.
```
Comment (Srivaralakshmi) [inline cicd/pipelines/uninstalling-pipelines.adoc:16]:
```suggestion
If you uninstall the Operator without removing the CRs of optional components, you can not remove them later.
```
```suggestion
If you uninstall the Operator without removing the CRs of optional components, you cannot remove them later.
```

File: modules/op-deleting-the-pipelines-component-and-custom-resources.adoc [hunk 2]
```diff
@@ -25,3 +25,8 @@ Deleting the CRs will delete the {pipelines-title} components, and all the Tasks
 ====
 
 . Click *Delete* to confirm the deletion of the CRs.
+
+[IMPORTANT]
+====
+Repeat the procedure to find and remove CRs of optional components such as {tekton-chains}, before uninstalling the Operator. If you uninstall the Operator without removing the CRs of optional components, you can not remove them later.
```
Comment (Srivaralakshmi) [inline modules/op-deleting-the-pipelines-component-and-custom-resources.adoc:31]:
```suggestion
Repeat the procedure to find and remove CRs of optional components such as {tekton-chains}, before uninstalling the Operator. If you uninstall the Operator without removing the CRs of optional components, you cannot remove them later.
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@sounix000 Left a few suggestions. PTAL. Thanks!

**Review by Srivaralakshmi — APPROVED** [review]
@sounix000 LGTM :)

**Review by vdemeester — APPROVED** [review]

**Pull request merged**